include/swift/Basic/Features.def

```diff
@@ -464,6 +464,10 @@ EXPERIMENTAL_FEATURE(AssumeResilientCxxTypes, true)
 /// Import inherited non-public members when importing C++ classes.
 EXPERIMENTAL_FEATURE(ImportNonPublicCxxMembers, true)
 
+/// Synthesize static factory methods for C++ foreign reference types and import
+/// them as Swift initializers.
+EXPERIMENTAL_FEATURE(GenerateCxxRefTypeInit, true)
```

Comment:
I think we can come up with a slightly better name here. From a developer's point of view, the `Generate` part is an implementation detail of the compiler, so I'd say we can remove it from the name. How about something like `ForeignReferenceTypeInitializers`?